recipes/fastgltf/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
```

Comment:
the `test_v1_package` is no longer required for new recipes: https://github.com/conan-io/conan-center-index/discussions/17406

But you can keep as it does not harm and does not affect the CI.

Reply:
I didn't add it myself, so I might as well just remove it.